sdk/rust/serde_wormhole/Cargo.toml

```diff
@@ -1,17 +1,21 @@
 [package]
 name = "serde_wormhole"
-version = "0.1.0"
-edition = "2021"
 description = "Serde data format for VAA payloads"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
 
 [dependencies]
-base64 = "0.13"
-itoa = "1.0.1"
-serde = { version = "1.0.103", default-features = false }
-serde_bytes = "0.11.5"
-thiserror = "1.0"
+base64.workspace = true
+itoa.workspace = true
+serde.workspace = true
```

Comment:
Is this a downgrade of the serde version?

Reply:
When publishing a crate, the highest version will be fetched to validate that the library can be built. So when this package is used for something in our repo like the smart contracts, it is just requiring that any version fetched is okay as long as it is major version 1.

The serde patches do not break compatibility with previous versions as far as I can tell... Those libraries have been on 1.0 for quite a while. But I can set the version to 1.0 to be extra safe. How do you feel about this?

Reply:
If that is the standard, I don't feel it's unreasonable to comply, and then I suppose it comes down to the integrator's `Cargo.lock` anyway.